juju/bakeryjs
# Migrate to TypeScript

## Done:
- Add TypeScript
- Migrate module and tests to TypeScript
- Update module build.

## QA:
- `yarn clean`
- `yarn build`
- `yarn link`
- in another project (e.g. https://github.com/canonical/jaas-dashboard/) run `yarn link "@canonical/macaroon-bakery"` and then `yarn install`.
- Check that there are no import errors and you can still authenticate using macaroons.

Fixes: #48.
